docs/en/serverless/inventory.mdx

```diff
@@ -0,0 +1,98 @@
+---
+slug: /serverless/observability/inventory
+title: Inventory
+description: Learn about the new Inventory experience that enables you to monitor all your entities from one single place.
+tags: [ 'serverless', 'observability', 'inventory' ]
+---
+
+import Roles from './partials/roles.mdx'
+
+<p><DocBadge template="technical preview" /></p>
+
+Inventory provides a single place where you can observe the status of your entire ecosystem of hosts, containers and services at a glance, even just from logs. From there, you can monitor and understand the health of your entities, check what needs attention and start your investigations. 
+
+<DocCallOut title="Note">
+   The new Inventory requires the Elastic Entity Model (EEM). To learn more, refer to <DocLink slug="/serverless/observability/elastic-entity-model" />.
+</DocCallOut>
+
+<DocImage url="./images/inventory-catalog.png" alt="Inventory catalog"/>
+
+Inventory is currently available for hosts, containers and services, but it will scale to support all of your entities.
+
+The Elastic Entity Model currently supports the inventory experience (as identified by `host.name`, `service.name`, and `container.id`) located in data identified by the following index patterns:
+
+**Hosts**
+
+Where `host.name` is set in `metrics-*`, `logs-*`, `filebeat-*`, and `metricbeat-*`
+
+**Services**
+
+Where `service.name` is set in `filebeat*`, `logs-*`, `metrics-apm.service_transaction.1m*`, and `metrics-apm.service_summary.1m*`
+  
+**Containers**
+
+Where `container.id` is set in `metrics-*`, `logs-*`, `filebeat-*`, and `metricbeat-*`
+
+Inventory allows you to:
+
+- Filter for your entities to provide a high-level view of what you have leveraging your own tags and labels
+- Drill down into any host, container or services to help you understand performance
+- Debug resource bottlenecks with your service caused by their containers and the hosts they run on.
+- Easily discover all entities related to the host, container or service you are viewing by leveraging your tags/labels
+
+## Explore your entities
+
+1. In your ((observability)) project, go to **Inventory** to view all of your entities.
+
+    When you open the Inventory for the first time, you need to enable the Elastic Entity Model to start using it. Once enabled, the Inventory will be accessible to anyone with the appropriate privileges.
```

Comment:
Is this something that will be obvious to users? Will it be like an empty state page that tells them they need to enable EEM? If so, could we say:
```suggestion
    When you open the Inventory for the first time, you'll be asked to enable the EEM. Once enabled, the Inventory will be accessible to anyone with the appropriate privileges.
```